mybatis-plus/runtime/src/main/java/com/baomidou/mybatisplus/core/toolkit/reflect/SpringReflectionHelper.java

```diff
@@ -0,0 +1,8 @@
+package com.baomidou.mybatisplus.core.toolkit.reflect;
+
+public class SpringReflectionHelper {
+
+    public static Class<?>[] resolveTypeArguments(Class<?> clazz, Class<?> genericIfc) {
+        throw new RuntimeException("Illegal Access is class in quarkus.");
```

Comment:
It's a little bit confuse about `Illegal Access is class in quarkus.` What you mean is NOT ALLOWED?

Reply:
this class only used with native complie，should not be run at runtime

Reply:
OK, in this case, can I suggest to
```suggestion
        throw new RuntimeException("Not Supported");
```